### PR TITLE
cmd/kubeadm/app: fix staticcheck warning

### PR DESCRIPTION
**Pull request by hwdef**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

fix staticcheck warning

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Review by SataQiu — COMMENTED** [review]
Thanks
/lgtm
/approve

**Comment by ereslibre** [comment]
Thank you @hwdef!

/priority backlog
/lgtm
/approve
/hold
^ cc/ @neolit123 @rosti @fabriziopandini @yastij for double checking

**Review by rosti — COMMENTED** [review]
@hwdef these look interesting. What kind of static analyzer do you use?

File: cmd/kubeadm/app/cmd/alpha/certs.go [hunk 1]
```diff
@@ -32,7 +32,6 @@ import (
 	"k8s.io/kubernetes/cmd/kubeadm/app/cmd/options"
 	cmdutil "k8s.io/kubernetes/cmd/kubeadm/app/cmd/util"
 	"k8s.io/kubernetes/cmd/kubeadm/app/constants"
-	kubeadmconstants "k8s.io/kubernetes/cmd/kubeadm/app/constants"
```
Comment (neolit123) [inline cmd/kubeadm/app/cmd/alpha/certs.go]: i think there was a plan to enable a tool to catch double imports, but it hasn't made it in k/k yet.

**Comment by hwdef** [comment]
@rosti 
I use staticcheck in [golangci-lint](https://github.com/golangci/golangci-lint).

File: cmd/kubeadm/app/cmd/alpha/certs.go (hunk 1 above)
Comment (hwdef) [reply cmd/kubeadm/app/cmd/alpha/certs.go]: I want to know ,if I found other double imports,which one should I leave?
Just like the above, should I leave 34 line or 35 line?

Comment (neolit123) [reply cmd/kubeadm/app/cmd/alpha/certs.go]: my guess is that across the kubeadm code base we are more consistent using `kubeadmconstants`. this PR moved that in the other direction, but overall as long as the imports don't have conflicting (in meaning) names and if there are no double imports, it's OK.

Comment (hwdef) [reply cmd/kubeadm/app/cmd/alpha/certs.go]: I understand, thank you very much.

**Pull request merged**